web/modules/custom/dept_node/dept_node.module

```diff
@@ -127,6 +127,12 @@ function dept_node_preprocess_node(array &$variables) {
   // Add the topic banner to the output, if relevant.
   dept_node_topic_banner($variables);
 
+  // Exit early if it's autocomplete - see dept_search.module
+  // preprocess_node for handling of autocomplete results.
+  if ($view_mode === 'search_autocomplete') {
```

Comment:
Any reason not to have this before the dept_node_topic_bannner call?

Reply:
Agreed, this would be better. Good spot! I'll move this up